src/main/java/io/mosip/mimoto/controller/V2IssuersController.java

```diff
@@ -100,14 +101,12 @@ public ResponseEntity<Object> getCredentialsSupportedForIssuer(@PathVariable("is
         return ResponseEntity.status(HttpStatus.OK).body(responseWrapper);
     }
 
-    @PostMapping(value = "/get-token/issuer", consumes = {MediaType.APPLICATION_FORM_URLENCODED_VALUE})
-    public ResponseEntity getToken(@RequestParam Map<String, String> params, @RequestParam String issuer) {
+    @PostMapping(value = "/get-token/{issuer}", consumes = {MediaType.APPLICATION_FORM_URLENCODED_VALUE})
```

Comment:
is this an existing API of mimoto or new api?

Reply:
refactored this to use the existing API